### ISSUE TITLE
The client.send_message() dont works

### ISSUE THREAD

**Issue by HakiBl4ck** [D4rkwat3r/ProjZ#3]
i try execute this code, but dont work, i send messages to the bot account and he dont answer me, pls how to i fix it?

**Comment by D4rkwat3r** [comment]
Can you give me any project z account so that I can provoke this error? You can create a new one, I can't do that.

**Comment by HakiBl4ck** [comment]
Okey, i understand. But the Lib is outdated?

**Comment by D4rkwat3r** [comment]
> Okey, i understand. But the Lib is outdated?

Yes, the library is outdated, it has not been updated for a very long time. However, most of the functions are still working, the main problem is with send_message, as I have not been able to find the cause of the error. That's why I need an account to check it out in more detail.

**Comment by HakiBl4ck** [comment]
Well, when I try to run my bot, it prints that the device is not supported and
"Check if You have the latest version of the app"
